.env.example

```diff
@@ -112,6 +112,8 @@ NEXT_PUBLIC_METADATA_SCHEMA=0xd00c966351896bd3dc37d22017bf1ef23165f859d7546a2aba
 NEXT_PUBLIC_EAS_CONTRACT_ADDRESS=0x4200000000000000000000000000000000000021
 NEXT_PUBLIC_EAS_SCHEMA_REGISTRY_ADDRESS=0x4200000000000000000000000000000000000020
 
+# The address of a deployed MACI instance
+NEXT_MACI_ADDRESS=""
```

Comment:
Should this also be added to env.js?

Reply:
Not necessary, but recommended, because it can verify if it's configured correctly before running the app via a Zod schema. For example:
```ts
NEXT_PUBLIC_ADMIN_ADDRESSES: z.string().startsWith("0x"),
```